cli/tests/test_git_remotes.rs

```diff
@@ -562,32 +562,11 @@ fn test_git_remote_with_global_git_remote_config() {
     ");
 
     let output = work_dir.run_jj(["git", "remote", "rename", "foo", "bar"]);
-    // Divergence from Git: we read the remote from the global
```

Comment:
this change means the divergence no longer exists, and so I have updated this test